### PR TITLE
Introduction fo the (EU) Energy label

### PR DESCRIPTION
**Pull request by Corniel**
Specially due to the `+`-es, this is preferable over an enum.

### CODE REVIEW

File: src/Qowaiv/Conversion/Sustainability/EnergyLabelTypeConverter.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using Qowaiv.Sustainability;
+
+namespace Qowaiv.Conversion.Sustainability;
+
+/// <summary>Provides a conversion for a Sex.</summary>
```
Comment (LauraXiulan) [inline src/Qowaiv/Conversion/Sustainability/EnergyLabelTypeConverter.cs:5]: Wrong summary

File: specs/Qowaiv.Specs/Sustainability/EnergyLabel_specs.cs [hunk 2]
```diff
@@ -0,0 +1,521 @@
...
+        var formatted = Svo.EnergyLabel.ToString("SomeFormat", FormatProvider.CustomFormatter);
+        Assert.AreEqual("Unit Test Formatter, value: 'A++', format: 'SomeFormat'", formatted);
+    }
+
+    [TestCase(null, "A++", "A++")]
+    [TestCase("U", "B", "B")]
+    [TestCase("l", "G", "g")]
+    [TestCase("l", "A++", "a++")]
+    [TestCase("L", "A++", "a++")]
+    public void fomrat_dependent(string format, EnergyLabel svo, string formatted)
```
Comment (LauraXiulan) [inline specs/Qowaiv.Specs/Sustainability/EnergyLabel_specs.cs:275]: format

File: specs/Qowaiv.Specs/Sustainability/EnergyLabel_specs.cs [hunk 3]
```diff
@@ -0,0 +1,521 @@
...
+    public void can_be_sorted_using_compare()
+    {
+        var sorted = new[]
+        {
+            default,
+            default,
+            EnergyLabel.F,
+            EnergyLabel.C,
+            EnergyLabel.A(),
+            EnergyLabel.Unknown,
```
Comment (LauraXiulan) [inline specs/Qowaiv.Specs/Sustainability/EnergyLabel_specs.cs:305]: Unknown is always highest? I would have expected it after the default

Comment (Corniel) [reply specs/Qowaiv.Specs/Sustainability/EnergyLabel_specs.cs:335]: default/Empty comes direct before G, so that would be strange.

**Pull request merged**